test/test_sparse_csr.py

```diff
@@ -1206,6 +1211,140 @@ def test_sparse_csr_unary_inplace(self, device, dtype, op):
             self.assertEqual(actual.col_indices(), expect.col_indices())
             self.assertEqual(actual._nnz(), expect._nnz())
 
+    def _test_pow_check_error(self, func, device, dtype, is_out=False):
+        # First ensure that the op fails for exponent as zero
+        inp = make_tensor((1, 2), dtype=dtype, device=device).to_sparse_csr()
+        if is_out:
+            out = self.genSparseCSRTensor(inp.size(), inp._nnz(),
+                                          device=device, dtype=dtype,
+                                          index_dtype=inp.crow_indices().dtype)
+
+        # Sparse CSR only supports 2D tensors as inputs
+        # Fail early to prevent silent success with this test
+        assert inp.ndim == 2, "Expected 2D input tensor for Sparse CSR"
+
+        with self.assertRaisesRegex(RuntimeError, "Exponent must be greater than 0 for Sparse CSR Layout."):
+            # Generate exponent as 0 per the given dtype
+            if dtype.is_complex:
+                exp = 0 + 0j
+            elif dtype.is_floating_point:
+                exp = 0.
+            else:
+                exp = 0
```

Comment:
Might be worth it to add a couple tests for negative exponents as well?

Reply:
Makes sense. Thanks for pointing out, Mario.